src/status.js

```diff
@@ -0,0 +1,32 @@
+export function saveLocal(list) {
+  window.localStorage.setItem('localTasks', JSON.stringify(list));
+}
+
+export function status(elem, list) {
+  list.forEach((task) => {
+    if (task === elem) {
+      task.isCompleted = !task.isCompleted;
+    }
+  });
```

Comment:
-[OPTIONAL] I noticed you are using `forEach` and object comparison to find elements in an array. I think using the `find function` where you can compare the `id` of each object makes your code look simpler and reduce the number of lines. What do you think about that?